react/components/CategoryTranslation/CategoryTranslation.tsx

```diff
@@ -32,39 +63,123 @@ const CategoryTranslation: FC = () => {
 
   const { id, ...categoryInfo } = entryInfo?.category || ({} as Category)
 
+  const downloadCategories = () => {
+    setHasError(false)
+    setDownloading(true)
+    fetchCategories({
+      variables: { active: onlyActive, locale: selectedLocale },
+    })
+  }
+
+  useEffect(() => {
+    // eslint-disable-next-line vtex/prefer-early-return
+    if (data) {
+      parseJSONToXLS(data.categoryTranslations, {
+        fileName: 'category-data',
+        sheetName: 'category_data',
+      })
```

Comment:
@filafb Is it possible to have the fileName be a construct of 'category-data'+locale?
so if I click on it-IT my file will be named category-data-it-IT etc